pkg/plugin/sdk/deployment.go

```diff
@@ -688,3 +688,32 @@ func (o StageStatus) toModelEnum() model.StageStatus {
 		return model.StageStatus_STAGE_FAILURE
 	}
 }
+
+type StageCommand struct {
+	Commander string
+	Type      CommandType
+}
+
+type CommandType int32
+
+const (
+	CommandType_APPROVE_STAGE CommandType = iota
+	CommandType_SKIP_STAGE
+)
```

Comment:
We use PascalCase for naming.

```suggestion
const (
	CommandTypeApproveStage CommandType = iota
	CommandTypeSkipStage
)
```

Reply:
Thank you, I fixed it on [e95e38b](https://github.com/pipe-cd/pipecd/pull/5639/commits/e95e38b58188396c37bf68ab7cc4789881ff2438)